app/views/layouts/_main_nav.html.erb

```diff
@@ -1,26 +1,27 @@
-<% default_nav_links = NavigationLink.default_section.ordered.to_a %>
-<% other_nav_links = NavigationLink.other_section.ordered.to_a %>
```

Comment:
This would be Russian doll caching which should be an optimization but in this cause I think it would be a negligible effect. We can overlook it for now.

Reply:
After looking at it again, I want to point out that these two cache are not directly related to each other and may not clear together when you expect them to